build/libxml2/patches/series

```diff
@@ -1,6 +0,0 @@
-CVE-2016-5131.patch
```

Comment:
If there are no patches remaining, just remove the entire patches/ directory and the empty series file.

Reply:
done